.github/workflows/release.yml

```diff
@@ -51,7 +51,7 @@ jobs:
       run: ./gradlew dependencies
 
     - name: Build release APK and bundle
-      run: ./gradlew :app:assembleRelease :app:bundleRelease
+      run: ./gradlew :app:assembleFreeRelease :app:bundleFreeRelease
```

Comment:
I'm not very sure about this. For reproducibility reasons, I upload the binary generated here to the Play Store as well, so not having the NonFree variant is a dealbreaker. I also don't think that the paths to generated binaries are correct here so that's gonna need fixing as well.